app/views/transfer/new.html.erb

```diff
@@ -173,3 +177,9 @@
       });
     </script>
   <% end %>
+
+  <% if Flipflop.enabled?(:maintenance_mode) %>
+    <script type="text/javascript">
+      $(':input').prop('disabled', true);
```

Comment:
Adding
`$('#new_transfer').hide();`
in addition to disabling feels like it would be slightly less confusing to someone that stumbled across this. Thoughts?

Reply:
Yeah, that's a good suggestion. I don't see any downside to hiding the form as well.